api/src/pcapi/core/subscription/api.py

```diff
@@ -412,3 +412,10 @@ def start_workflow(
     pcapi_repository.repository.save(user)
     fraud_check = fraud_api.start_fraud_check(user, thirdparty_id, content)
     return fraud_check
+
+
+def stop_workflow(
+    user: users_models.User, thirdparty_id: str, content: fraud_models.DMSContent
+) -> fraud_models.BeneficiaryFraudCheck:
+    fraud_check = fraud_api.mark_fraud_check_failed(user, thirdparty_id, content, [])
```

Comment:
Quand tu passes une constante (et non une variable nommée), ça peut valoir le coup de préciser l'argument pour être plus clair. Ici, ça donnerait : 

```suggestion
    fraud_check = fraud_api.mark_fraud_check_failed(user, thirdparty_id, content, reasons=[])
```